libs/common/src/sql-db-entities/base-entities/auditable.base-entity.test.ts

```diff
@@ -0,0 +1,81 @@
+import { AuditableBaseEntity } from '.';
+import { DbRequestSource } from '../helpers';
+
+// We cannot create instances of abstract classes, so we define a test class below
+class TestAuditableBaseEntity extends AuditableBaseEntity {}
+
+describe('AuditableBaseEntity', () => {
+  describe('updateLastUpdatedBy', () => {
+    describe("when the request source platform is 'PORTAL'", () => {
+      const userId = 'abc123';
+
+      it.each`
```

Comment:
I normally view it.each tests as being different inputs triggering different outputs rather than here where you are checking one update of an auditableBaseEntity having different values set correctly. Unless there's a benefit I'm missing can we change this to just be a single test asserting the whole object matches the object object we expect?
partly I also want the test to check that's all we're adding here and nothing else  which this method misses out on